modules/upgrade/pages/proxy-upgrade.adoc

```diff
@@ -0,0 +1,50 @@
+[[update.suse.manager.proxy]]
+= Update {productname} Proxy
```

Comment:
What's the difference between an 'update', an 'upgrade' and a 'migration'?

I think we need consistent naming. Now the file is `proxy-upgrade`, but this title is `Update`, and then bellow we talk about `updates`.

Checking the server (https://github.com/SUSE/doc-susemanager/blob/f07fff0d73a909dd29bb42b2cfe60a4a082f824f/modules/upgrade/pages/server-upgrade.adoc), it seems we have the same problem.

So FMPOV: let's choose one convention (updating and migrating, or upgrading and migrating), and then let's use it everywhere.

Reply:
I intentionally used the "wrong" filename here as well.  Yes, later we should change both filenames at once.

Reply:
ACK.